app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java

```diff
@@ -44,12 +47,12 @@ protected void onCreate(Bundle savedInstanceState) {
 
         selectedSpeaker = dbSingleton.getSpeakerbySpeakersname(speaker);
 
-
         TextView biography = (TextView) findViewById(R.id.speaker_bio);
         ImageView linkedin = (ImageView) findViewById(R.id.imageView_linkedin);
         ImageView twitter = (ImageView) findViewById(R.id.imageView_twitter);
         ImageView github = (ImageView) findViewById(R.id.imageView_github);
         ImageView fb = (ImageView) findViewById(R.id.imageView_fb);
+        FloatingActionButton shareButton=(FloatingActionButton) findViewById(R.id.share_speaker_button);
```

Comment:
why the share is a floating action button. Mostly share is a top right button in the navigation bar.